app/graphql/types/profile.rb

```diff
@@ -118,13 +123,8 @@ def system_id(args)
     end
 
     def latest_test_result_batch(args)
-      ::RecordLoader.for(
-        ::TestResult,
-        column: :host_id,
-        where: { profile_id: object.id },
-        includes: :rule_results, # TODO: check for lookaheads
-        order: 'created_at DESC'
-      ).load(system_id(args))
+      ::RecordLoader.for(::TestResult, column: :host_id, where: { profile_id: object.id },
+                                       includes: :rule_results, order: 'created_at DESC').load(system_id(args))
```

Comment:
Seems like unrelated change.

Reply:
It is a related change, we have some rubocop limits :grin: